webview/src/experiments/components/table/body/branchDivider/styles.module.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../../../shared/variables.scss';
+
+.branchName {
+  padding: 40px 20px 10px;
```

Comment:
[C] The next problem we will have to solve is: "how can a user compare two separate records on different branches". For me, this means we need a mechanism to display them directly above/below one another. Maybe we could allow pinning or we could start with something like "show only selected" as a multi-select option.